geometry_msgs/Point[] L_Y_cones
geometry_msgs/Point[] L_B_cones
geometry_msgs/Point[] R_Y_cones
geometry_msgs/Point[] R_B_cones
sensor_msgs/Image left_frame
sensor_msgs/Image right_frame


